tests/view/renderer.js

```diff
@@ -918,26 +920,6 @@ describe( 'Renderer', () => {
 			expect( domSelection.getRangeAt( 0 ).collapsed ).to.equal( true );
 		} );
 
-		it( 'should not add ranges if different editable is selected', () => {
```

Comment:
100%CC is just a help for us to understand whether some case wasn't missed by test. Here, you clearly add a feature and write 0 tests for it which means that if this feature breaks in the future (or disappears) there's a high chance that it won't be noticed by tests.

So don't head for 100%CC but for tests which validate feature correctness. Then you'll have 100%CC as well.

BTW, manual test is missing as well. There are clear cases like dragging from right to left which were broken. They should work now. I propose to create a single, selection rendering test for it and check multiple scenarios there.